hibernate-reactive-core/src/test/java/org/hibernate/reactive/annotations/README.md

```diff
@@ -0,0 +1,45 @@
+These tests verify the new annotations intended to replace the **DBSelectionExtenstion** which only allows enabling/disabling some or all the tests within a test class.
+
+This extension feature didn't allow enabling/disabling at the test method level.
+
+The **RunDBType** and **SkipDBType** annotations are based off these ORM annotations and supporting utilities
+* [RequiresDialect](https://github.com/hibernate/hibernate-orm/blob/main/hibernate-testing/src/main/java/org/hibernate/testing/RequiresDialect.java)
+* [SkipForDialect](https://github.com/hibernate/hibernate-orm/blob/main/hibernate-testing/src/main/java/org/hibernate/testing/SkipForDialect.java)
```

Comment:
There's no need for this in the readme.
This should be mentioned in the JavaDoc

Reply:
Done